src/jvmMain/kotlin/Main.kt

```diff
@@ -265,7 +258,11 @@ class TBCSim : CliktCommand() {
         val specFilter = specFilterStr?.split(",")
         val categoryFilter = categoryFilterStr?.split(",")
 
-        if (calcEP) {
+        if (calcEP && configFile?.exists() == true) {
```

Comment:
Sorry about this, looks like I broke that option a long time ago and never fixed the docs...  The EP rankings on the site mostly replaced the intent of that feature.  

Would the most useful output for a single-character sim be just a table of results to the console?  Or, would something else be better?

Reply:
I found creating the list of my personal EPs to be the most useful thing. Arcane mages have a few different points where haste, in particular, changes very dramatically in value. Understanding where I am relative to those points of inflection is what I wanted to get, and the current output with my local changes does that.